src/assets/variables.scss

```diff
@@ -9,8 +9,9 @@
  * - list max width: text width + utils width + 4 + 8 (paddings) = 1250px
  */
 $messages-text-max-width: calc(70 * var(--default-font-size));
-$message-utils-width: calc(52px + 4 * var(--default-grid-baseline) + 120px);
-$messages-list-max-width: calc($messages-text-max-width + $message-utils-width + 3 * var(--default-grid-baseline));
+$message-avatar-width: calc(32px + 4 * var(--default-grid-baseline));
```

Comment:
```suggestion
$message-avatar-width: calc(var(--avatar-size) + 4 * var(--default-grid-baseline)); 
```

Reply:
We don't have that in CSS.. But I'll leave a comment in variables stylesheet